client/event_container_opening.go

```diff
@@ -0,0 +1,55 @@
+package client
+
+import (
+	"github.com/regner/albiondata-client/lib"
+	"github.com/regner/albiondata-client/log"
+	"github.com/regner/albiondata-client/notification"
+)
+
+type eventContainerOpening struct {
+	Slots []int64 `mapstructure:"1"`
+}
+
+func (event eventContainerOpening) Process(state *albionState) {
+	log.Debug("Got container opening event...")
+
+	if state.LocationId == 0 {
+		log.Warn("The players location has not yet been set. Please transition zones so the location can be identified.")
```

Comment:
Assuming both these take the same type of parameter can you define this once as msg and pass that to log and notification?

Reply:
done